osu.Game/Screens/Edit/Editor.cs

```diff
@@ -1137,6 +1138,62 @@ private void cancelExit()
             loader?.CancelPendingDifficultySwitch();
         }
 
+        public void SeekAndSelectHitObjects(string timestamp, Action<LocalisableString> onError)
```

Comment:
Probably would just call it `HandleTimestamp` at this point. Too specific to be named anything else.

Also the `onError` thing shouldn't exist really - there's not much point in marshaling the `NotificationsOverlay` through `OsuGame`. `INotificationsOverlay` is already available in `Editor`; just use that directly.

Reply:
Renamed and moved Notifications inside